services/src/unit-test/java/uk/gov/moj/sdt/services/SubmitQueryServiceTest.java

```diff
@@ -196,57 +180,48 @@ public void testSubmitQueryRequestOutage() {
 
         final IGlobalParameter receiveTimeOutParam = new GlobalParameter();
-        receiveTimeOutParam.setName("TARGET_APP_RESP_TIMEOUT");
+        receiveTimeOutParam.setName(TARGET_APP_RESP_TIMEOUT);
         receiveTimeOutParam.setValue("12000");
-        EasyMock.expect(this.mockGlobalParamCache.getValue(IGlobalParameter.class, "TARGET_APP_RESP_TIMEOUT"))
-                .andReturn(receiveTimeOutParam);
+        when(this.mockGlobalParamCache.getValue(IGlobalParameter.class, TARGET_APP_RESP_TIMEOUT))
+                .thenReturn(receiveTimeOutParam);
 
         final IGlobalParameter maxQueryReq = new GlobalParameter();
         maxQueryReq.setName("MCOL_MAX_CONCURRENT_QUERY_REQ");
         maxQueryReq.setValue("5");
-        EasyMock.expect(this.mockGlobalParamCache.getValue(IGlobalParameter.class, "MCOL_MAX_CONCURRENT_QUERY_REQ"))
-                .andReturn(maxQueryReq);
+        when(this.mockGlobalParamCache.getValue(IGlobalParameter.class, "MCOL_MAX_CONCURRENT_QUERY_REQ"))
+                .thenReturn(maxQueryReq);
 
         final IGlobalParameter contactDetails = new GlobalParameter();
         contactDetails.setName("CONTACT_DETAILS");
         contactDetails.setValue("SDT Team");
-        EasyMock.expect(this.mockGlobalParamCache.getValue(IGlobalParameter.class, "CONTACT_DETAILS")).andReturn(
+        when(this.mockGlobalParamCache.getValue(IGlobalParameter.class, "CONTACT_DETAILS")).thenReturn(
                 contactDetails);
 
         final OutageException outageEx = new OutageException("OUTAGE_ERROR", "Server unavailable.");
-        this.mockConsumerGateway.submitQuery(submitQueryRequest, 1000, 12000);
-        EasyMock.expectLastCall().andThrow(outageEx);
+        mockConsumerGateway.submitQuery(submitQueryRequest, 1000, 12000);
+
+        doThrow(outageEx).when(mockConsumerGateway).submitQuery(submitQueryRequest, 1000, 12000);
 
         final IErrorMessage errorMsg = new ErrorMessage();
         errorMsg.setErrorCode("TAR_APP_ERROR");
         errorMsg.setErrorDescription("Server unavailable.");
         errorMsg.setErrorText("The system encountered a problem.");
 
-        EasyMock.expect(this.mockErrorMsgCacheable.getValue(IErrorMessage.class, "TAR_APP_ERROR")).andReturn(
+        when(this.mockErrorMsgCacheable.getValue(IErrorMessage.class, "TAR_APP_ERROR")).thenReturn(
                 errorMsg);
 
-        EasyMock.replay(mockConsumerGateway);
-        EasyMock.replay(mockGlobalParamCache);
-        EasyMock.replay(mockErrorMsgCacheable);
-        EasyMock.replay(mockBulkCustomerDao);
-
         SdtContext.getContext().setRawInXml("criteria");
         this.submitQueryService.submitQuery(submitQueryRequest);
 
-        EasyMock.verify(mockConsumerGateway);
-        EasyMock.verify(mockGlobalParamCache);
-        EasyMock.verify(mockErrorMsgCacheable);
-        EasyMock.verify(mockBulkCustomerDao);
+        assertEquals(null, SdtContext.getContext().getRawOutXml(),"Raw output xml should be null");
```

Comment:
Could use assertNull() here instead of assertEquals().

Reply:
done